tests/PHPStan/Analyser/data/array-column-php7.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace ArrayColumn;
+
+use function PHPStan\Testing\assertType;
+
+function test(array $array): void
+{
+	/** @var array<int, array{column: string, key: array}> $array */
+	assertType('array<int, string>', array_column($array, 'column', 'key'));
+	/** @var array<int, array{column: string, key: array|string}> $array */
```

Comment:
I don't like `@var` in these tests. Please write a class + a method (different namespaces for php7/php8) and type these arrays as `@param`.

Reply:
I just pushed a new version. Is there any desired structure for the namespaces in tests, since it's not PSR-4?

Reply:
> I just pushed a new version. Is there any desired structure for the namespaces in tests

There isn't, there's a `classmap` entry in composer.json :)